jvb/src/main/kotlin/org/jitsi/videobridge/relay/RelayMessageTransport.kt

```diff
@@ -323,6 +323,9 @@ class RelayMessageTransport(
         }
     }
 
+    override fun webSocketError(cause: Throwable) =
+        logger.error("Failed to establish colibri websocket: ${cause.message}")
```

Comment:
I don't think this happens only on failure to establish - it looks like other things can trigger it (like invalid websocket framing).  Which probably means we want to log for errors on endpoint websockets, too - and maybe do something to tear down already-established ones?